### PR TITLE
Add ExpenseIt! Gmail add-ons solution

### PR DESCRIPTION
**Pull request by asrivas**

### CODE REVIEW

File: expenses-add-on/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Store user values using Google Apps Script's [Properties Service][propertiesservice].
+
+[cardservice]: https://developers.google.com/apps-script/reference/card-service/
+[propertiesservice]:https://developers.google.com/apps-script/reference/properties/
+
+## Try it
+
+1. Create a copy of the Gmail Add-on code by clicking this [link][code]. This will create a new 
+script in your G Suite Developer Hub where you can manage your Apps Script projects. 
+1. Select View > Show manifest file, which opens a file titled `appsscript.json`.
```
[the remark this answers is not in the document]
Comment (asrivas) [reply expenses-add-on/README.md]: Noted for the style guide and fixed -- thank you!

File: expenses-add-on/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+1. Select Publish > Deploy from manifest.
+1. Next to the entry Latest Version (Head) Click Get ID.
```
[the remark this answers is not in the document]
Comment (asrivas) [reply expenses-add-on/README.md]: done

File: expenses-add-on/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+1. Open the [Gmail add-on settings][settings] tab.
+1. In the Add-ons tab, ensure that you have selected the Enable developer add-ons for my account checkbox.
+1. Paste your add-on's deployment ID into the Developer add-ons textbox and click Install.
+1. In the Install developer add-on dialog that appears, click the checkbox to
+indicate that you trust this developer (yourself), then click Install.
+1. Open any message in Gmail, either on web or mobile.
+1. A new button that resembles a receipt ![icon](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/expenses-add-on/receipt-icon.png)
+should appear on the right or bottom of your screen, depending on your platform.
+1. Click the button to authorize the add-on and follow the prompts.
+1. Open an email and fill out the receipt information and click `New Sheet`.
```
Comment (sqrrrl) [inline expenses-add-on/README.md]: `New Sheet` -> **New Sheet**.

File: expenses-add-on/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+1. Create a copy of the Gmail Add-on code by clicking this [link][code]. This will create a new 
+script in your G Suite Developer Hub where you can manage your Apps Script projects. 
+1. Select **View** > **Show manifest file**, which opens a file titled `appsscript.json`.
+1. Select **Publish** > **Deploy from manifest**.
+1. Next to the entry **Latest Version (Head)** click **Get ID**.
+1. Select and copy the Deployment ID value.
+1. Close the dialog and the Deployments window.
+1. Open the [Gmail add-on settings][settings] tab.
+1. In the Add-ons tab, ensure that you have selected the Enable developer add-ons for my account checkbox.
+1. Paste your add-on's deployment ID into the Developer add-ons textbox and click Install.
```
Comment (sqrrrl) [inline expenses-add-on/README.md]: Button text should be formatted like menus and other labels. Here and a few lines below.

File: expenses-add-on/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+1. Create a copy of the Gmail Add-on code by clicking this [link][code]. This will create a new 
+script in your G Suite Developer Hub where you can manage your Apps Script projects. 
+1. Select **View** > **Show manifest file**, which opens a file titled `appsscript.json`.
+1. Select **Publish** > **Deploy from manifest**.
+1. Next to the entry **Latest Version (Head)** click **Get ID**.
+1. Select and copy the Deployment ID value.
+1. Close the dialog and the Deployments window.
+1. Open the [Gmail add-on settings][settings] tab.
+1. In the Add-ons tab, ensure that you have selected the Enable developer add-ons for my account checkbox.
```
Comment (sqrrrl) [inline expenses-add-on/README.md]: Format UI label here.

File: expenses-add-on/README.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+## Try it
+
+1. Create a copy of the Gmail Add-on code by clicking this [link][code]. This will create a new 
+script in your G Suite Developer Hub where you can manage your Apps Script projects. 
+1. Select **View** > **Show manifest file**, which opens a file titled `appsscript.json`.
+1. Select **Publish** > **Deploy from manifest**.
+1. Next to the entry **Latest Version (Head)** click **Get ID**.
+1. Select and copy the Deployment ID value.
+1. Close the dialog and the Deployments window.
+1. Open the [Gmail add-on settings][settings] tab.
```
Comment (sqrrrl) [inline expenses-add-on/README.md]: Format UI label.

File: expenses-add-on/README.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+[propertiesservice]:https://developers.google.com/apps-script/reference/properties/
+
+## Try it
+
+1. Create a copy of the Gmail Add-on code by clicking this [link][code]. This will create a new 
+script in your G Suite Developer Hub where you can manage your Apps Script projects. 
+1. Select **View** > **Show manifest file**, which opens a file titled `appsscript.json`.
+1. Select **Publish** > **Deploy from manifest**.
+1. Next to the entry **Latest Version (Head)** click **Get ID**.
+1. Select and copy the Deployment ID value.
```
Comment (sqrrrl) [inline expenses-add-on/README.md]: Format UI label.

File: expenses-add-on/README.md [hunk 8]
```diff
@@ -29,18 +29,18 @@ script in your G Suite Developer Hub where you can manage your Apps Script proje
...
+1. Select and copy the **Deployment ID** value.
 1. Close the dialog and the Deployments window.
-1. Open the [Gmail add-on settings][settings] tab.
-1. In the Add-ons tab, ensure that you have selected the Enable developer add-ons for my account checkbox.
-1. Paste your add-on's deployment ID into the Developer add-ons textbox and click Install.
-1. In the Install developer add-on dialog that appears, click the checkbox to
-indicate that you trust this developer (yourself), then click Install.
+1. Open the **[Gmail add-on settings][settings]** tab.
+1. In the Add-ons tab, ensure that you have selected the **Enable developer add-ons for my account** checkbox.
+1. Paste your add-on's deployment ID into the Developer add-ons textbox and click **Install**.
```
Comment (sqrrrl) [inline expenses-add-on/README.md]: One more -- **Developer add-ons**

File: expenses-add-on/README.md [hunk 9]
```diff
@@ -29,18 +29,18 @@ script in your G Suite Developer Hub where you can manage your Apps Script proje
 1. Select **View** > **Show manifest file**, which opens a file titled `appsscript.json`.
 1. Select **Publish** > **Deploy from manifest**.
 1. Next to the entry **Latest Version (Head)** click **Get ID**.
-1. Select and copy the Deployment ID value.
+1. Select and copy the **Deployment ID** value.
 1. Close the dialog and the Deployments window.
```
Comment (sqrrrl) [inline expenses-add-on/README.md]: **Deployments**

File: expenses-add-on/README.md (hunk 8 above)
Comment (asrivas) [reply expenses-add-on/README.md]: d'oh

File: expenses-add-on/README.md (hunk 9 above)
Comment (asrivas) [reply expenses-add-on/README.md]: good catch -- thank you

**Pull request merged**